src/store/actions/clients.js

```diff
@@ -0,0 +1,83 @@
+import { Toast } from '../../config/Notifications';
+import * as actionTypes from './actionTypes';
+
+import { getAllClients, createClient } from '../../services/clientService';
+
+const _fetchAllClients = () => {
+  return {
+    type: actionTypes.FETCHING_CLIENTS,
+  };
+};
+
+const _fetchAllClientsFailed = error => {
+  return {
+    type: actionTypes.FETCHING_CLIENTS_FAILURE,
+    error,
+    resultCount: 0,
+  };
+};
+
+const _receiveAllClients = (clients, count) => {
+  return {
+    type: actionTypes.FETCHING_CLIENTS_SUCCESS,
+    clients,
+    resultCount: count,
+  };
+};
+
+const _postNewClient = () => {
+  return {
+    type: actionTypes.CREATING_NEW_CLIENT,
+  };
+};
+
+const _postNewClientFailed = error => {
+  return {
+    type: actionTypes.CREATING_NEW_CLIENT_FAILURE,
+    error,
+  };
+};
+
+const _postNewClientSuccess = (client) => {
```

Comment:
for consistency lets remove brackets around `(client)` :)

Reply:
Bracket removed.